cli/app.go

```diff
@@ -1496,6 +1497,11 @@ After creation, use 'viam module update' to push your new module to app.viam.com
 					Name:  "generate",
 					Usage: "generate a new modular resource via prompts",
 					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:  moduleFlagLanguage,
+							Usage: "resource type to use in module",
```

Comment:
Update this usage text

Reply:
Ah, a relic of copying and pasting the work from another PR. Thanks for catching!